contrib/features/karaf/pom.xml

```diff
@@ -0,0 +1,128 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Hm, I see using that directory hierarchy we need do duplicate that definition (the whole POM). That's not nice.
Would it make sense to use a `contrib` directory below `features` and `bom` instead one contrib directory?
I don't know, lets discuss...

Reply:
I would prefer that.